otx/algorithms/segmentation/adapters/mmseg/models/heads/mixin.py

```diff
@@ -23,42 +23,21 @@
 # pylint: disable=abstract-method, unused-argument, keyword-arg-before-vararg
 
 
-class SegmentOutNormMixin(nn.Module):
-    """SegmentOutNormMixin."""
-
-    def __init__(self, *args, enable_out_seg=True, enable_out_norm=False, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        self.enable_out_seg = enable_out_seg
-        self.enable_out_norm = enable_out_norm
-
-        if enable_out_seg:
-            if enable_out_norm:
-                self.conv_seg = AngularPWConv(self.channels, self.out_channels, clip_output=True)
-        else:
-            self.conv_seg = None
-
-    def cls_seg(self, feat):
-        """Classify each pixel."""
-        if self.dropout is not None:
-            feat = self.dropout(feat)
-        if self.enable_out_norm:
-            feat = normalize(feat, dim=1, p=2)
-        if self.conv_seg is not None:
-            return self.conv_seg(feat)
-        return feat
-
-
-class AggregatorMixin(nn.Module):
-    """A class for creating an aggregator."""
+class Mixin(nn.Module):
```

Comment:
I renamed it to `SegMixinModule`. Unfortunately, we can't merge CustomOTXHead with SegMixinModule since CustomOTXHead should be inherited from base head class from mmseg on the go. It should be done in class factory or in function. I changed function to class factory with callback function. It looks more appropriate I think